packages/react-charts/src/components/ChartTheme/themes/base-theme.ts

```diff
@@ -149,46 +150,43 @@ export const BaseTheme = {
     ...LAYOUT_PROPS,
     style: {
       axis: {
-        fill: chart_axis_axis_Fill.value,
+        fill: chart_axis_axis_Fill.var,
         strokeWidth: chart_axis_axis_stroke_Width.value,
-        stroke: chart_axis_axis_stroke_Color.value,
+        stroke: chart_axis_axis_stroke_Color.var,
         strokeLinecap: STROKE_LINE_CAP,
         strokeLinejoin: STROKE_LINE_JOIN
       },
       axisLabel: {
         ...LABEL_CENTERED_PROPS,
         padding: chart_axis_axis_label_Padding.value,
-        stroke: chart_axis_axis_label_stroke_Color.value
+        stroke: chart_axis_axis_label_stroke_Color.var
       },
       grid: {
-        fill: chart_axis_grid_Fill.value,
+        fill: chart_axis_grid_Fill.var,
         stroke: 'none',
         pointerEvents: chart_axis_grid_PointerEvents.value,
         strokeLinecap: STROKE_LINE_CAP,
         strokeLinejoin: STROKE_LINE_JOIN
       },
       ticks: {
-        fill: chart_axis_tick_Fill.value,
+        fill: chart_axis_tick_Fill.var,
         size: chart_axis_tick_Size.value,
-        stroke: chart_axis_tick_stroke_Color.value,
+        stroke: chart_axis_tick_stroke_Color.var,
         strokeLinecap: STROKE_LINE_CAP,
         strokeLinejoin: STROKE_LINE_JOIN,
         strokeWidth: chart_axis_tick_Width.value
       },
-      tickLabels: {
-        ...LABEL_PROPS,
-        fill: chart_axis_tick_label_Fill.value
-      }
```

Comment:
Not sure this `fill` should be removed? 

The `fill` had a value of #4f5255. Not sure what `chart_global_label_Fill.va`r is?